packages/driver/cypress/e2e/commands/navigation.cy.js

```diff
@@ -765,7 +765,6 @@ describe('src/cy/commands/navigation', () => {
       })
 
       cy.visit('fixtures/redirection-loop-a.html')
-      cy.get('div').should('contain', 'this should fail?')
```

Comment:
Previously removing this line would allow the test to not throw the failure. I made a couple of changes below to now not require this line.